shared/actions/tracker.js

```diff
@@ -621,6 +631,26 @@ function getTracking (username: string): Promise {
   })
 }
 
+function fillFolders (getState: () => {favorite: FavoriteState}, username: string): TrackerActionCreator {
+  return (dispatch, getState) => {
+    const root = getState().favorite.folders || {}
+    const pubIg = root.public.ignored || []
+    const pubTlf = root.public.tlfs || []
+    const privIg = root.private.ignored || []
+    const privTlf = root.private.tlfs || []
+
+    const tlfs = [].concat(pubIg, pubTlf, privIg, privTlf).filter(f => f.users.filter(u => u.username === username).length)
```

Comment:
we look at all folders and include them if they contain the user. keeping ignored, that's correct right @malgorithms